orga/app/components/organization-participant/list.js

```diff
@@ -2,8 +2,11 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
 import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
 
 export default class List extends Component {
+  @tracked shouldShowDeletionModal = false;
```

Comment:
**suggestion** : Je trouve le should un peu superflu et que showDeletionModal est suffisament porteur de sens 😄